qiskit/pulse/schedule.py

```diff
@@ -249,6 +250,60 @@ def filter(self, *filter_funcs: List[Callable],
             time_ranges: Time intervals to keep, e.g. [(0, 5), (6, 10)]
             intervals: Time intervals to keep, e.g. [Interval(0, 5), Interval(6, 10)]
         """
+        total_filter = self._construct_filter(*filter_funcs,
```

Comment:
For some reason, I feel like `composed_filter` would have been more clear to me on first read. I don't feel strongly though, it's fine if you leave it as `total_filter`.